mobile/src/main/java/com/google/samples/apps/iosched/MainApplication.kt

```diff
@@ -33,33 +28,6 @@ import javax.inject.Inject
 class MainApplication : Application() {
 
     // Even if the var isn't used, needs to be initialized at application startup.
-    @Inject lateinit var analyticsHelper: AnalyticsHelper
-
-    override fun onCreate() {
-        // ThreeTenBP for times and dates, called before super to be available for objects
-        AndroidThreeTen.init(this)
-
-        // Enable strict mode before Dagger creates graph
-        if (BuildConfig.DEBUG) {
-            enableStrictMode()
-        }
-        super.onCreate()
-
-        if (BuildConfig.DEBUG) {
-            Timber.plant(Timber.DebugTree())
-        } else {
-            Timber.plant(CrashlyticsTree())
-        }
-    }
-
-    private fun enableStrictMode() {
-        StrictMode.setThreadPolicy(
-            Builder()
-                .detectDiskReads()
-                .detectDiskWrites()
-                .detectNetwork()
-                .penaltyLog()
-                .build()
-        )
-    }
+    @Inject
+    lateinit var analyticsHelper: AnalyticsHelper
```

Comment:
Can this have its own initializer?

Reply:
Sure let me blend `AppStartup` with `Hilt` :)